libs/sdk-ui-tests-e2e/cypress/tools/api.ts

```diff
@@ -53,4 +53,46 @@ export class Api {
             req.headers["Authorization"] = `Bearer ${userToken}`;
         });
     }
+
+    postDrillDownAttribute = (drillFromAttribute: string, payload?: any) => {
+        if (getBackend() !== "BEAR") {
```

Comment:
Don't need to check backend, this method will be used in  `setUpDrillDownAttribute` only, and that method is already check the backend